src/main/java/uk/gov/pay/webhooks/message/EventMapper.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.pay.webhooks.webhookevent;
```

Comment:
Package name doesn't match filename. Is this intentional or can you rename one or t'other?

Reply:
👍